building/build-debs/homeworld-admin-tools/src/verify.py

```diff
@@ -90,7 +98,7 @@ def check_ssh_with_certs(hostname=None):
     del env["SSH_AGENT_PID"]
     keypath = access.renew_ssh_cert()
     try:
-        result = subprocess.check_output(["ssh", "-i", keypath, "root@%s.%s" % (hostname, config.external_domain), "echo confirmed"], env=env)
+        result = subprocess.check_output(["ssh", "-o", "StrictHostKeyChecking=yes", "-i", keypath, "root@%s.%s" % (hostname, config.external_domain), "echo confirmed"], env=env)
```

Comment:
The SSH parameters are repeated a lot everywhere in special cases like this one; maybe it would be good to make them a global variable.